<launch>   
    <!-- Node 1 -->
    <!-- <node pkg="foma" type="ceiling_camera_node.py" name="ceiling_camera" output="screen"/> -->

    <!-- Node 2 -->
    <node pkg="foma" type="fish_detection_node.py" name="fish_detection_node" output="screen"/>

    <!-- Node 3 -->
    <node pkg="foma" type="gui_node.py" name="gui_node" output="screen"/>

    <!-- Node 4 -->
    <node pkg="foma" type="localization_node.py" name="localization_node" output="screen"/>

    <!-- Node 5 -->
    <node pkg="foma" type="light_dimmer_node.py" name="light_dimmer_node" output="screen"/>

    <!-- Node 6 -->
    <!-- <node pkg="foma" type="dummy_video_camera_node.py" name="dummy_video_camera_node" output="screen"/> -->
</launch>